ansible/playbooks/setup-kubernetes.yml

```diff
@@ -0,0 +1,26 @@
+---
+# Copyright 2024-Present, Rackspace Technology, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- hosts: localhost
+  become: True
+  gather_facts: "{{ gather_facts | default(true) }}"
+  environment: "{{ deployment_environment_variables | default({}) }}"
+  vars_files:
+    - 'vars/default.yml'
+  roles:
+    - { role: "k8s_install",
```

Comment:
no need for the exterior braces here.

Reply:
My ansible wan't happy

`  roles:
    - role: "k8s_install",
                         ^ here`